### ISSUE TITLE
Upgrade to 2.2.1 with PGSQL fails:  column "anonymous" contains null values  

### ISSUE THREAD

**Issue by kfrohwein** [sulu/sulu#5601]
| Q | A
| --- | ---
| Sulu Version | 2.2.1

# Actual Behavior

```
doctrine:schema:update --force   

 Updating database schema...


In AbstractPostgreSQLDriver.php line 51:
                                                                                                
  An exception occurred while executing 'ALTER TABLE se_roles ADD anonymous BOOLEAN NOT NULL':  
                                                                                                
  SQLSTATE[23502]: Not null violation: 7 ERROR:  column "anonymous" contains null values        
                                                                                                

In PDOConnection.php line 91:
                                                                                          
  SQLSTATE[23502]: Not null violation: 7 ERROR:  column "anonymous" contains null values  
                                                                                          

In PDOConnection.php line 86:
                                                                                          
  SQLSTATE[23502]: Not null violation: 7 ERROR:  column "anonymous" contains null values  
                                                                                          

```

# Expected Behavior

Schema can be updated

# Steps to Reproduce

Run update with pgsql

# Possible Solutions

I guess a default has to be set on a NULL column or pgsql will fail. Will see if I can make a pull request tomorrow.

**Comment by alexander-schranz** [comment]
@kfrohwein Thank you for report, the boolean should not be `nullable` so that is correct but it should have a `DEFAULT` value to false, so we need to set something like we did here:

https://github.com/sulu/sulu/blob/3f2225e3defcc8cfc218380c2e65d65b1084bf31/src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/User.orm.xml#L15-L17

This should be added to: https://github.com/sulu/sulu/blob/3f2225e3defcc8cfc218380c2e65d65b1084bf31/src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/Role.orm.xml#L14

Do you want to create a PR for it?

**Comment by niklasnatter** [comment]
Thanks for reporting! Fixed by #5605.
